### PR TITLE
Academyv2 changesv3

### PR DESCRIPTION
**Pull request by CryptoManion**
updates on images

### CODE REVIEW

File: docs/architecture/layer-0.md [hunk 1]
```diff
@@ -153,12 +153,9 @@ Today, there are a handful of notable L0 networks that are ushering in a new era
...
         <td><strong>Polkadot</strong> is a blockchain protocol that unites an entire network of purpose-built
-            blockchains known as ‘parachains’.
-            <p>
-                <strong>Parachains</strong> are connected together and secured by
-            </p>
-            <p>
-                the Polkadot Relay Chain. They can also connect with external networks via bridges.
+            blockchains known as ‘parachains’.           
+                <strong>Parachains</strong> are connected together and secured by the Polkadot Relay Chain. 
+              They can also connect with external networks via bridges.
```
Comment (yurikoinaba) [inline docs/architecture/layer-0.md:158]:
```suggestion

               <p><strong>Parachains</strong> are connected together and secured by the Polkadot Relay Chain. 
              They can also connect with external networks via bridges.
```

File: docs/wallets/cold-wallets.md [hunk 2]
```diff
@@ -41,13 +41,13 @@ Hot Wallets and Cold Wallets.
 
 These wallets allow you to instantly deploy assets unto dapps by using your private key to authorize your funds to be spent by the dapps smart contract.
 
-Metmask, Coinbase wallet and Trust wallet are all examples of hot wallets that are always connected to the internet.
+Metamask, Coinbase wallet and Trust wallet are all examples of hot wallets that are always connected to the internet.
```
Comment (yurikoinaba) [inline docs/wallets/cold-wallets.md:44]: `MetaMask`

File: docs/architecture/layer-0.md (hunk 1 above)
Comment (yurikoinaba) [reply docs/architecture/layer-0.md:158]: Add the `<p>` at the beginning before `<strong>`

**Comment by yurikoinaba** [comment]
@Haxen2000 Can you help Manon take a look at these errors

```


09:02:09.886 | SyntaxError: /opt/buildhome/repo/docs/architecture/layer-0.md: Expected corresponding JSX closing tag for <td>. (182:0)
-- | --
09:02:09.886 | 180 \| </tr>
09:02:09.886 | 181 \| `}</code></pre>
09:02:09.886 | > 182 \| </table>
09:02:09.887 | \| ^
09:02:09.887 | 183 \| <h2 {...{"id":"what-makes-horizen-the-top-choice-for-l0s"}}>{`What Makes Horizen the Top Choice for L0s?`}</h2>
09:02:09.887 | 184 \| <h3 {...{"id":"competitive-edge-in-privacy-technologies"}}>{`Competitive Edge in Privacy Technologies`}</h3>
09:02:09.887 | 185 \| <p>{`As a zero-knowledge enabled network of blockchains, Horizen is leveraging the latest privacy technologies to enhance the utility and scalability of blockchain networks.`}</p>
09:02:09.887 | Error: Image static/img/bitcoin-glossary/visual-bitcoin-transaction-diagram.jpeg used in docs/cryptocurrency/bitcoin-glossary.md not found.
09:02:09.887 | at async Promise.all (index 1)
09:02:09.887 | Error: Image static/img/ethereum-glossary/what-is-evm.jpeg used in docs/cryptocurrency/ethereum-glossary.md not found.
09:02:09.887 | at async Promise.all (index 0)
09:02:09.888 | Error: Image static/img/ZBF_article_graphics_2_how-amms-work.jpg used in docs/defi/automated-market-maker-amms.md not found.
09:02:09.888 | at async Promise.all (index 0)
09:02:09.888 | Error: Image static/img/blockchain-oracles/ZBF-academy-Decentralized-oracle-graphic.jpg used in docs/defi/blockchain-oracles.md not found.
09:02:09.888 | at async Promise.all (index 2)
09:02:09.888 | Error: Image static/img/ethereum-glossary/what-is-evm.jpeg used in docs/interoperability/ethereum-virtual-machine-evm.md not found.
09:02:09.888 | at async Promise.all (index 0)
09:02:09.888 | [ERROR] Client bundle compiled with errors therefore further build is impossible.
09:02:09.965 | Failed: build command exited with code: 1
09:02:10.852 | Failed: error occurred while running build command


```

**Review by Haxen2000 — APPROVED** [review]

**Pull request merged**